resources/ext/cli/delete-reports.erb

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -e
+
+pdb_service=<%= EZBake::Config[:user] %>
+pdb_db_name="$pdb_service"
+<% if EZBake::Config[:build_type] == "pe" %>
+pg_user=pe-postgres
+jar_resource=puppetlabs/puppetdb/sql/delete_reports.sql
+<% else %>
+pg_user=postgres
+jar_resource=sql/delete_reports.sql
+<% end %>
+pdb_jar=/opt/puppetlabs/server/apps/puppetdb/puppetdb.jar
+
+function printerr {
+  echo "$@" 1>&2
+}
+
+function usage {
+  cat <<USAGE
+Usage: puppetdb delete-reports [OPTIONS]
+    --service SERVICE     the puppetdb service name (default: $pdb_service)
+    --db-name NAME        the puppetdb database name (default: $pdb_db_name)
+    --pg-user USER        the postgres system user (defult: $pg_user)"
+    --resource RESOURCE   the jar resource to extract (default: $jar_resource)
+    --jar PATH            the full path to the puppetdb jar file"
+                            (default: $pdb_jar)"
+USAGE
+}
+
+function misuse {
+  usage 1>&2
+  exit 2
+}
+
+if [ "$(id -u)" -ne 0 ]; then
+  printerr "Please run as root"
+  exit 2
+fi
+
+while test $# -gt 0; do
+  case "$1" in
+    --service)
+      shift
+      test $# -gt 0 || misuse
+      pdb_service="$1"
+      shift
+      ;;
+    --db-name)
+      shift
+      test $# -gt 0 || misuse
+      pdb_db_name="$1"
+      shift
+      ;;
+    --pg-user)
+      shift
+      test $# -gt 0 || misuse
+      pg_user="$1"
+      shift
+      ;;
+    --resource)
+      shift
+      test $# -gt 0 || misuse
+      jar_resource="$1"
+      shift
+      ;;
+    --jar)
+      shift
+      test $# -gt 0 || misuse
+      pdb_jar="$1"
+      shift
+      ;;
+    -h|--help)
+      usage
+      exit 0
+      ;;
+    *)
+        misuse
+  esac
+done
+
+printerr "Stopping puppetdb..."
+service "$pdb_service" stop
+printerr "Stopped puppetdb."
+
+tmp_dir="$(mktemp -d)"
+(cd "$tmp_dir"
+  if command -v unzip >/dev/null; then
+    printerr "Using 'unzip' to extract archive"
+    unzip $pdb_jar "$jar_resource" 1>&2
+  elif command -v jar >/dev/null; then
+    printerr "Using 'jar' to extract archive"
+    jar -xf $pdb_jar "$jar_resource"
+  else
+    printerr "Could not locate 'unzip' or 'jar' on your PATH"
```

Comment:
Just checking a 2019.2.2 system I happened to have lying around, neither unzip or jar were installed by default.  We ship a copy of jar in the `pe-java8-devel rpm` (`pe-java-devel` in older releases).  Perhaps we could install that as part of this script to ensure that jar was available at `/opt/puppetlabs/server/apps/java/lib/jvm/java/bin/jar`.  

Short of that, we could have the error message suggest that the customer install that RPM themselves.